dashboard/app/controllers/pd/workshop_enrollment_controller.rb

```diff
@@ -45,6 +45,8 @@ def create
       render :full
     else
       @enrollment = ::Pd::Enrollment.new workshop: @workshop
+      @enrollment.build_school_info school_info_params
```

Comment:
Yes, I was using that before.  Because I'm using accepts_nested_attributes_for, using build_ here creates the new school_info item, but it's not saved until we save this parent pd_enrollment, which I guess is good because it means we save everything at once, after they all pass validations (I'm hoping).  I've not yet found a similar helper to build_ which is clever enough to use an existing school_info, though maybe there is a pattern.